src/styles/blocks/stars--3.css

```diff
@@ -0,0 +1,44 @@
+.stars--3 {
+  display: flex;
+  margin-top: 4px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). It seems you're using `margin-top` here, but ensure that you're not using `margin-bottom` on the same elements elsewhere to avoid potential margin collapse.